src/components/balances/AssetsTable/index.tsx

```diff
@@ -137,6 +147,8 @@ const AssetsTable = ({
     [hiddenAssets, balances.items, showHiddenAssets],
   )
 
+  const hasNoAssets = balances.items.length === 1 && balances.items[0].balance === '0'
```

Comment:
Not sure if there is a better way to find this out. Any ideas?

Reply:
I think this is the only way as our endpoint always returns the native token balance. Even if it is 0.